scripts/gcs/a6000/install.sh

```diff
@@ -0,0 +1,52 @@
+# setup ubuntu
+apt-get update
+apt-get upgrade -y
+apt-get dist-upgrade -y
+
+apt-get update
+apt-get upgrade -y
+apt-get dist-upgrade -y
+
+# get apps
+apt-get install git -y
+
+# get pip
+cd ~/Downloads/
+
+wget https://bootstrap.pypa.io/get-pip.py
+python get-pip.py
+rm get-pip.py
+
+# install gphoto
+wget https://raw.githubusercontent.com/gonzalo/gphoto2-updater/master/gphoto2-updater.sh && chmod +x gphoto2-updater.sh && sudo ./gphoto2-updater.sh
+
+# create service file
+touch a6000.service
+
+echo "[Unit]" >> a6000.service
+echo "Description=A6000 Camera Picture Take and Serve" >> a6000.service
+echo "" >> a6000.service
+echo "[Service]" >> a6000.service
+echo "ExecStart=/home/$USER/Desktop/SUAS-Competition/scripts/gcs/a6000/startup_picture_serve_and_take.sh" >> a6000.service
+echo "User=odroid" >> a6000.service
```

Comment:
Why don't you just create a service file, and then move it with this script? That would take a lot less work than creating the service file, adding all of the information, etc.